packages/calcite-components/src/components/functional/Validation.tsx

```diff
@@ -3,26 +3,32 @@ import { h, LuminaJsx } from "@arcgis/lumina";
 import { Scale, Status } from "../interfaces";
 import { IconNameOrString } from "../icon/interfaces";
 
-interface ValidationProps extends LuminaJsx.HTMLAttributes {
+interface ValidationProps extends LuminaJsx.CustomAttributes {
   scale: Scale;
   status: Status;
   icon?: IconNameOrString | boolean;
   id?: string;
   message: string;
+  ref?: (el: HTMLDivElement) => void;
 }
 
 export const CSS = {
   validationContainer: "validation-container",
 };
 
+const setValidationReference = (referenceElement: HTMLDivElement): HTMLDivElement => {
+  return referenceElement;
+};
+
 export const Validation = ({
   scale,
   status,
   id,
   icon,
   message,
+  ref,
 }: ValidationProps): TemplateResult => (
-  <div class={CSS.validationContainer}>
+  <div class={CSS.validationContainer} ref={ref ? ref : setValidationReference}>
```

Comment:
@anveshmekala Can you explain what this is doing? It doesn't appear to be storing the ref element anywhere or doing anything with it.

Reply:
This to avoid TypeError when `ref` is not set.  When user toggles the visibility of Validation element, ref is updated which fails when set to `undefined`.

To encounter this, we are parsing a dummy method which imitates a reference handler.

Reply:
Why is the function returning the reference element? Shouldn't it be a void function?

Reply:
Agree, it should return `void`. 

```
const setValidationReference = (): void => {
  return;
}
```

how about updating the name to `validationReference` instead of `setValidationReference` ?

Reply:
Yeah that sounds good